code/modules/paperwork/folders.dm

```diff
@@ -8,36 +8,88 @@
 
 	autoignition_temperature = 522 // Kelvin
 	fire_fuel = 1
+	var/crayon = null
+
+/obj/item/weapon/folder/New()
+	..()
+	update_icon()
+
+/obj/item/weapon/folder/black
+	crayon = "black"
 
 /obj/item/weapon/folder/blue
-	desc = "A blue folder."
-	icon_state = "folder_blue"
+	crayon = "blue"
 
 /obj/item/weapon/folder/red
-	desc = "A red folder."
-	icon_state = "folder_red"
+	crayon = "red"
+
+/obj/item/weapon/folder/white
+	crayon = "sterile"
 
 /obj/item/weapon/folder/yellow
-	desc = "A yellow folder."
-	icon_state = "folder_yellow"
+	crayon = "yellow"
 
-/obj/item/weapon/folder/white
-	desc = "A white folder."
-	icon_state = "folder_white"
+/obj/item/weapon/folder/purple
+	crayon = "purple"
+
+/obj/item/weapon/folder/orange
+	crayon = "orange"
+
+/obj/item/weapon/folder/green
+	crayon = "green"
+
+/obj/item/weapon/folder/rainbow
+	crayon = "rainbow"
 
 /obj/item/weapon/folder/update_icon()
 	overlays.len = 0
 	if(contents.len)
 		overlays += image(icon = icon, icon_state = "folder_paper")
+
+	switch(crayon)
```

Comment:
better way:
`icon_state = "folder_[crayon]"`

rename the icon state of the medbay one to `folder_sterile`
rename the default one to `folder_`
rename the clown one to `folder_rainbow`

Reply:
Done. Made sure Deity's inhand sprites play nice with the folders, too.